.ci/jobs/apm-agent-java-load-mbp.yml

```diff
@@ -0,0 +1,44 @@
+---
+##### MBP JOB
+- job:
+    name: apm-agent-java/apm-agent-java-load-mbp
+    display-name: APM Agent Java Load Testing
+    description: APM Agent Java Load Testing
+    script-path: .ci/load/Jenkinsfile
+    project-type: multibranch
+    scm:
+        - github:
+            branch-discovery: no-pr
+            discover-pr-forks-strategy: merge-current
+            discover-pr-forks-trust: permission
+            discover-pr-origin: merge-current
+            discover-tags: false
+            disable-pr-notifications: true
+            notification-context: 'apm-ci'
+            head-filter-regex: '(master|PR-.*)'
```

Comment:
Filter only master and PRs

Reply:
Since the repository is the agent repository, not all PRs are related to load-testing.
- could we have a way to filter those, maybe on branch/PR name pattern ?

For example, if we want to filter on branch name, would having `(master|wip-load-testing-.*)` instead allow to have:
- one instance of the job for `master`
- one instance of the job for each branch matching pattern `wip-load-testing-*`.

One very minor downside of relying on branch name and not on PR would be to require pushing to the upstream repository.

Reply:
Let me explain a bit better:

The requirement we had in mind was to support the run of PRs that are related to the load testing, for such we took in consideration:
- there will no be automated builds for any PRs, or commits into the master branch.
- The only way to interact with it would be manually -> [apm-agent-java-load-mbp](https://apm-ci.elastic.co/job/apm-agent-java/job/apm-agent-java-load-mbp/) (once the job is created)